services/report/parser/types.py

```diff
@@ -41,41 +38,31 @@ class ParsedRawReport(object):
 
     def __init__(
         self,
-        toc: Any,
-        env: Any,
+        toc: list[str],
         uploaded_files: list[ParsedUploadedReportFile],
         report_fixes: Any,
     ):
         self.toc = toc
-        self.env = env
         self.uploaded_files = uploaded_files
         self.report_fixes = report_fixes
 
-    def has_toc(self) -> bool:
-        return self.toc is not None
-
-    def has_env(self) -> bool:
-        return self.env is not None
-
     def has_report_fixes(self) -> bool:
         return self.report_fixes is not None
 
     @property
     def size(self):
         return sum(f.size for f in self.uploaded_files)
 
-    def content(self) -> BytesIO:
-        buffer = BytesIO()
-        if self.has_toc():
-            for file in self.get_toc():
-                buffer.write(f"{file}\n".encode("utf-8"))
-            buffer.write(b"<<<<<< network\n\n")
-        for file in self.uploaded_files:
-            buffer.write(f"# path={file.filename}\n".encode("utf-8"))
-            buffer.write(file.contents)
-            buffer.write(b"\n<<<<<< EOF\n\n")
-        buffer.seek(0)
-        return buffer
+
+class LegacyParsedRawReport(ParsedRawReport):
```

Comment:
This doesn't read a `as_readable` method?

Reply:
Indeed, the only usage of this is here:
https://github.com/codecov/worker/blob/6ba6bba83d4d4ea93a491f1c902c741b3124d7ab/services/processing/processing.py#L100-L102